scripts/rook.sh

```diff
@@ -2,7 +2,8 @@
 
 ROOK_VERSION=${ROOK_VERSION:-"v1.6.2"}
 ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
-ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph"
+ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/"
+ROOK_DEPLOYMENT_PATH="cluster/examples/kubernetes/ceph"
```

Comment:
The path for the example manifests moved in 1.8, perhaps this will help:
```suggestion
ROOK_DEPLOYMENT_PATH="deploy/examples"
```

Reply:
Oh, I see you already changed this another place below. 
Where is the operator log so we can see if there are errors getting the cluster setup? At a glance, it seems at least a couple tests failed from basic rook setup, and csi logs might help with other tests that are failing to create volumes.

Reply:
@travisn operator logs are here https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/upgrade-tests-cephfs/detail/upgrade-tests-cephfs/4449/pipeline/99/. I tested this change locally everything is working fine not sure whats wrong.

Reply:
we don't use CSI provided by Rook to do any testing, we are making use of Rook to create a ceph cluster and will consume the ceph cluster from standalone cephcsi pods.

Reply:
I'm not seeing anything helpful in that log, it just seems the mon isn't forming quorum or the operator can't detect it. Is that test configured differently than the other tests that setup rook without an issue? Or it is worth rerunning to see if it was just a temporary failure for example with the CI network?

Reply:
nothing special here, the only change is this PR is changing the file paths as it's changed from 1.6.2 to 1.8.2, and the test is running fine locally. @travisn any other logs, we can check what is wrong here?